xbmc/video/VideoDatabase.h

```diff
@@ -592,19 +592,20 @@ class CVideoDatabase : public CDatabase
                            const std::map<std::string, std::string>& artwork,
                            int idShow,
                            int idEpisode = -1);
-  bool SetFileForEpisode(const std::string& fileAndPath, int idEpisode, int idFile);
-  bool SetFileForMovie(const std::string& fileAndPath, int idMovie, int idFile);
+  bool SetFileForEpisode(const std::string& fileAndPath, const int idEpisode, const int oldIdFile);
```

Comment:
Please drop `const` from the ints here. As ints are passed by value, a copy is always created, so const never affects anything outside this function, and as a result adds clutter to the external API without providing any real protection or optimization.